.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
@v3 is the latest

https://github.com/actions/checkout

Reply:
done